### ISSUE TITLE
media prompt type issue

### ISSUE THREAD

**Issue by leartgjoni-voiceflow** [actions-on-google/assistant-conversation-nodejs#7]
Following the example [here](https://developers.google.com/assistant/conversational/prompts-media#sample_code), this:
```
conv.add(new Media({
    mediaObjects: [
      {
        name: 'Media name',
        description: 'Media description',
        url: 'https://storage.googleapis.com/automotive-media/Jazz_In_Paris.mp3',
        image: {
          large: JAZZ_IN_PARIS_IMAGE,
        }
      }
    ],
    mediaType: 'AUDIO',
    optionalMediaControls: ['PAUSED', 'STOPPED'],
    startOffset: '2.12345s'
  }));
```
will throw a type error of: `Argument of type 'Media' is not assignable to parameter of type 'PromptItem'.`

Possible fix:
```
export type PromptItem =
  string |
  Simple |
  Content |
  Link |
  Suggestion |
  Canvas |
  OrderUpdate |
  Media
```

@taycaldwell

**Comment by nylsoo** [comment]
The documentation seems to use another actions library: (https://github.com/actions-on-google/actions-on-google-nodejs). To add media with this library you need to add it like this: 
```
conv.add(
    new Content({
      media: {
        mediaObjects: [
          {
            name: 'Media name',
            url: 'https://storage.googleapis.com/automotive-media/Jazz_In_Paris.mp3',
            image: { icon: { url: JAZZ_IN_PARIS_IMAGE, alt: `Jazz in paris image` } },
            mediaType: 'AUDIO',
            optionalMediaControls: ['PAUSED', 'STOPPED'],
            startOffset: '2.12345s',
          },
        ],
      },
    }),
  );
```

**Comment by Fleker** [comment]
I'm going to take a look at this, as Media is used in the [conversation-components sample](https://github.com/actions-on-google/actions-builder-conversation-components-nodejs), so it would in theory break that sample too.

I wouldn't advise you to mix those two libraries together, as they're meant to work only for their specific services.

**Comment by Fleker** [comment]
Seems like Image should also be exported.

**Comment by Fleker** [comment]
This was fixed in c0472b94eb7181ef59f6b2bf396edfd2f36b6a63.
